### DISCUSSION TITLE
Welcome to microsoft-365-community Discussions!

### DISCUSSION THREAD

**Discussion by dstrome** [MicrosoftDocs/microsoft-365-community#294] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HansHWestphal** [comment]
Hi,
Just completed the 2nd sharing is caring event with David and Hugo and was wondering if you would like a community left nav topic of Licensing. I could write this myself if you are interested.
my best,
HansHWestphal

**Reply by sympmarc** [reply]
Hi, @HansHWestphal. What type of licensing article(s) are you thinking of writing?

**Comment by HansHWestphal** [comment]
Hi
I could write just about every license in m365 with scenarios and Msrp costing... including full unattended bots running on m365 e3 unattended with azure windows virtual desktop

________________________________

**Reply by sympmarc** [reply]
@HansHWestphal - I think that would be a tough moving target. We do have one article about Power Platform licensing because it's so darn confusing. I think trying to track the pricing would be near impossible over time. 

What might be useful is some information about why you might choose different licenses from a business perspective. More of a qualitative approach than a quantitative one, with links to the appropriate pricing page(s). Make sense?
